src/nvim/buffer_defs.h

```diff
@@ -958,6 +958,11 @@ typedef struct {
   bool noautocmd;
   bool fixed;
   bool hide;
+/// cmdline_offset: Offset in units of "screen cell width" indicating how much
```

Comment:
no need for the quotes, cell width is a well known concept used everywhere already.

Reply:
Sure, but IIRC I copied this verbatim from nvim_open_win.